virtcontainers/fc.go

```diff
@@ -908,11 +908,42 @@ func (fc *firecracker) fcAddNetDevice(endpoint Endpoint) {
 	defer span.Finish()
 
 	ifaceID := endpoint.Name()
+
+	// The implementation of rate limiter is based on TBF.
+	// Rate Limiter defines a token bucket with a maximum capacity (size) to store tokens, and an interval for refilling purposes (refill_time).
+	// The refill-rate is derived from size and refill_time, and it is the constant rate at which the tokens replenish.
+	refillTime := int64(1000)
+	var rxRateLimiter models.RateLimiter
+	rxSize := fc.config.RxRateLimiter
+	if rxSize != int64(0) {
```

Comment:
a negative rate limit is valid ? :thinking:

Reply:
Hi @devimc  thanks!!!!! nice catch, I'll add negative check when parsing configs asap. ;)

Reply:
great! I think this should be `rxSize > int64(0) `

Reply:
okaly~~update asap. ;)